### PR TITLE
Updates for automated a11y testing

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- Changes some a11y related items found using automated tests

### CODE REVIEW

File: docs/_layouts/area-page.njk [hunk 1]
```diff
@@ -4,19 +4,6 @@ layout: default.njk
...
-      <div class="col-12 content">
-        {{ content | safe }}
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-12">
-        <hr class="minor mt-1 mb-6">
-      </div>
-    </div>
-  </div>
```
Comment (frogpad7) [inline docs/_layouts/area-page.njk:18]: If the area pages are not going to provide any text info above the cards, could we remove that info from the md files or is that going to be used elsewhere?

**Review by frogpad7 — CHANGES_REQUESTED** [review]
The deleted display from the area pages needs to be cleaned up, if that will not shhow up elsewhere.

File: docs/_layouts/area-page.njk (hunk 1 above)
Comment (isralduke) [reply docs/_layouts/area-page.njk:18]: that could be used elsewhere so we’re leaving it in the MD for now @frogpad7

**Review by frogpad7 — APPROVED** [review]
The deleted display from the area pages needs to be cleaned up, if that will not shhow up elsewhere.

File: docs/_layouts/area-page.njk (hunk 1 above)
Comment (frogpad7) [reply docs/_layouts/area-page.njk:18]: Sounds good; everything else works as I would expect it to.

**Pull request merged**